File: PLD1_InputProcessOutput/EX12_maleFemalePercentages/maleFemalePercentages.cpp
#include <iostream>
using namespace std;

int main()
{
    float male;
    float female;
    float totalStudent;
    float percentageMale;
    float percentageFemale;

    cout << "Enter number of males : ";
    cin >> male;
    cout << "Enter number of females : ";
    cin >> female;

    totalStudent = male + female;
    percentageMale = male / totalStudent * 100;
    percentageFemale = female / totalStudent * 100;

    cout << "Percentages of male student   : " << percentageMale << "%" << endl;
    cout << "Percentages of female student : " << percentageFemale << "%" << endl;

    return 0;
}

File: PLD1_InputProcessOutput/EX12_maleFemalePercentages/maleFemalePercentages.js
const prompt = require("prompt-sync")();

male = Number.parseFloat(prompt("Enter number of males : "));
female = Number.parseFloat(prompt("Enter number of males : "));

totalStudent = male + female;
percentageMale = (male / totalStudent) * 100;
percentageFemale = (female / totalStudent) * 100;

console.log(
  "Percentage Males student   : " + Number.parseInt(percentageMale) + "%"
);
console.log(
  "Percentage Females student : " + Number.parseInt(percentageFemale) + "%"
);
